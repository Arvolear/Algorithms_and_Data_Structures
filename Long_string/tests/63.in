1647429696998512795175391
1495094478865944831279142