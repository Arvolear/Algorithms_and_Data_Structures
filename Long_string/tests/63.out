3142524175864457626454533
152335218132567963896249
2463063044302272885290697315739948433524769994522
1